pallets/creator-staking/README.md

```diff
@@ -0,0 +1,29 @@
+# Creator Staking Pallet
+
+The Subsocial Creator Staking Pallet is a critical component of the Subsocial content creation ecosystem. It provides features for tracking staking information for creators by era, storing general info about total staking amounts per era, and managing the economic interactions between creators and their supporters.
+
+## Features
+
+- **Staking Information**: The pallet tracks staking information for creators by era, allowing users to stake funds to their favorite creators.
+
+- **Unstaking and Withdrawal**: Users can start the unbonding process, and once the unbonding period has passed, they can withdraw their staked funds.
+
+- **Reward Distribution**: The pallet calculates and distributes rewards for stakeholders and creators on a periodic basis (e.g., per era), incentivizing participation in the ecosystem.
+
+- **Lock Periods**: To prevent immediate withdrawals, the pallet manages lock periods. Stakers must wait out an unbonding period after initiating an unstaking operation.
```

Comment:
make sense just combine w/ "Unstaking and Withdrawal"

Reply:
same comment about using both "unbonding" and "unstaking"